asab/library/providers/zookeeper.py

```diff
@@ -300,13 +300,19 @@ async def list(self, path: str) -> list:
 		global_nodes = await self.Zookeeper.get_children(global_node_path) or []
 		global_items = await self.process_nodes(global_nodes, path)
 
+<<<<<<< HEAD
```

Comment:
??

Reply:
No! I am synchronising this PR  to latest master.